tools/d2g/d2g.go

```diff
@@ -502,7 +502,7 @@ func podmanEnvMappings(dwPayload *dockerworker.DockerWorkerPayload) string {
 
 	envVarNames := []string{}
 	for envVarName := range dwPayload.Env {
-		envVarNames = append(envVarNames, envVarName)
+		envVarNames = append(envVarNames, shell.Escape(envVarName))
```

Comment:
It looks like the env var name is being stored in its escaped form. I think it would be cleaner to store it in its raw form, but escape it when writing it the command.

Reply:
If `shell.Escape()` is called around this entire function as `shell.Escape(podmanEnvMappings(dwPayload))`, then it'll put single quotes around the ` -e ENV0 -e ENV1` list and podman won't interpret the flags properly at that point. The way I have it, only the `ENV0` or `ENV1` names will be shell escaped.

Reply:
Ah I see what you mean. I can escape inside of `podmanEnvSetting()`.